src/components/form/FieldBase.vue

```diff
@@ -108,6 +110,12 @@ export default {
     unlabelled() {
       return this.label === '';
     },
+    labelId() {
+      if (this.name && this.label) {
+        return `${kebabCase(this.name.trim())}-label`;
+      }
+      return null;
+    },
```

Comment:
I don't fully understand this computed property.

If we pass both the `name` and the `label`, then we trim the `name` and discard the `label`, right?

I think this could be simplified to:

```suggestion
    labelId() {
      return `${kebabCase(this.name.trim())}-label`;
    },
```

On one side, the label is only shown if the `label` prop is passed to the component.
We can consider that there will always be a `name` because it is marked as a required prop.

That's why I don't understand why we need to validate the presence of the `label` prop.

Reply:
I think the idea is to avoid having the `labelId` in the `aria-labelledby`, from `HdCheckbox`.

If that is the case, I would add a comment explaining it as it might be weird when revisiting this component in the future. As it is a single case, we don't have `aria-labelledby` in other inputs, it is a bit hard to reason the best structure for that. At the moment if fits our use case.